main.py

```diff
@@ -276,19 +294,27 @@ def performEvaluation(issue_data) -> Result:
     branch = issue_data[JsonKeys.BRANCH.value]
     commit_hash = issue_data[JsonKeys.COMMIT_HASH.value]
 
-    input_dir = create_issue_directory(issue_folder_dir, issue_id) # ../cf-12/input
-    clone_repository(url, input_dir)  # TODO: check if clonning is successful.
+    issue_folder_abs_dir = os.path.abspath(issue_folder_dir)
+    input_dir = create_issue_directory(issue_folder_abs_dir, issue_id)
+    clone_repository(url, input_dir) 
     repo_name = get_repository_name(url)
 
     if branch:
-        change_branch(branch, f"{input_dir}/{repo_name}")  
+        change_branch(branch, os.path.join(input_dir, repo_name))  
     
     if commit_hash:
-        checkout_commit(commit_hash, f"{input_dir}/{repo_name}")
-
-    specimin_command = build_specimin_command(repo_name, os.path.join(issue_folder_dir, issue_id), os.path.join(issue_folder_dir, specimin_project_name),issue_data[JsonKeys.ROOT_DIR.value], issue_data[JsonKeys.TARGETS.value])
-
-    result = run_specimin(issue_id ,specimin_command, os.path.join(issue_folder_dir, specimin_project_name))
+        checkout_commit(commit_hash, os.path.join(input_dir, repo_name))
+
+    specimin_command = ""
+    result: Result = None
+    specimin_path = get_specimin_env_var()
```

Comment:
This always use the local copy of Specimin. Why? Are you relying on the initial cloning process to set the environment variable? That's brittle, if so - it would be better here in `main.py` to do the following:
1. check for the environment variable. If it's present, use that.
2. check for a local copy in `resources/specimin`. If it's there, use that.
3. if none of the above, trigger a clone operation.

Reply:
Behavior: 
- if an environment variable exists, using that.
- checking a copy at `ISSUES/specimin`, if exists pulling a latest copy and using that.
- otherwise, cloning at "ISSUES/specimin"